src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -562,6 +562,43 @@ public void testForceOutput() {
         Assert.assertTrue(actualVC2.get(1).isPolymorphicInSamples());
     }
 
+    @Test
```

Comment:
Add a comment that this checks that the non-ref is removed even for multiple alt alleles

Reply:
ok